src/Glpi/Controller/IndexController.php

```diff
@@ -111,8 +112,22 @@ private function call(): void
         //Try to detect GLPI agent calls
         $rawdata = file_get_contents("php://input");
         if (!isset($_POST['totp_code']) && !empty($rawdata) && $_SERVER['REQUEST_METHOD'] === 'POST') {
+            //FIXME: how to do the same with controllers? The only way I see is a redirect
```

Comment:
Another solution would be not to allow inventory call on GLPI root.

Reply:
(I gues that would be preferable, existing include solution was quite a weird solution.)

Reply:
I prefer to keep the support on the index; this greatly eases the setup of the agent for admins.
We may ` forward` the request to the inventory controller, right?

Reply:
I don't know how to forward since this is a POST request.

Reply:
Should be OK with 74733ea736cf2a99834c782089fe9426e42ddab6